### PR TITLE
gotop: 4.0.1 -> 4.1.0

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change

https://github.com/xxxserxxx/gotop/releases/tag/v4.1.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/system/gotop/default.nix [hunk 1]
```diff
@@ -21,6 +21,7 @@ buildGoModule rec {
   meta = with lib; {
     description = "A terminal based graphical activity monitor inspired by gtop and vtop";
     homepage = "https://github.com/xxxserxxx/gotop";
+    changelog = "https://github.com/xxxserxxx/gotop/blob/v${version}/CHANGELOG.md";
     license = licenses.agpl3;
```
Comment (SuperSandro2000) [inline pkgs/tools/system/gotop/default.nix:25]: agpl3Plus or apgl3Only?

Comment (marsam) [reply pkgs/tools/system/gotop/default.nix]: oh, it's licensed with MIT.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 110933` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gotop</li>
  </ul>
</details>

**Comment by magnetophon** [comment]
I'm not using this anymore.
Anyone want to take over maintainership?

**Review by fabaff — APPROVED** [review]
- [x] package build (see https://github.com/NixOS/nixpkgs/pull/110933#issuecomment-768321367)
- [ ] all depending packages build
- [x] executables tested on x86_64

```bash
...ew/pr-110933]$ ./results/gotop/bin/gotop -V
2021/01/27 15:27:34 gopsutil reports Number of warnings: 1
2021/01/27 15:27:34 gopsutil reports Number of warnings: 1
gotop 0.0.0 (Hadean)
[...]
```

Looks good to me. We could consider to set the [current release](https://github.com/xxxserxxx/gotop/blob/master/cmd/gotop/main.go#L48) which would make `-V` more meaningful.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110933` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>gotop</li>
</ul>
</details>

**Pull request merged**